backend/Squirrel.Core/Squirrel.Core.WebAPI/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -32,11 +33,14 @@ public static void RegisterCustomServices(this IServiceCollection services)
         services.AddScoped<JwtIssuerOptions>();
         services.AddScoped<IJwtFactory, JwtFactory>();
         services.AddScoped<IAuthService, AuthService>();
+        services.AddScoped<IProjectService, ProjectService>();
         services.AddScoped<ITextService, TextService>();
     }
 
     public static void AddMongoDbService(this IServiceCollection services, IConfiguration configuration)
     {
+        services.AddAutoMapper(Assembly.GetAssembly(typeof(SampleProfile)));
+        services.AddAutoMapper(Assembly.GetAssembly(typeof(ProjectProfile)));
```

Comment:
you don't need this anymore, there is AddAutoMapper extension method from BLL project